### DISCUSSION TITLE
报错求助

### DISCUSSION THREAD

**Discussion by BasicSix** [hiroi-sora/Umi-OCR#214] [Q&A] [answered]
某天,用清理软件清理了下电脑,然后重启,发现启动不了UMI了,报错如下:

Unhandled exception in script

Failed to execute script 'main' due to unhandled exception: Can't find a usable init.tcl in the following directories:
{C:/Windows/TEMP/ME129402\tcl}
C:/Windows/TEMP/ ME129402/tcl8.6
C:/Windows/TEMP/fib/tcl8.6
D:/绿色软件/lib/tcl8.6 
D:/lib/tcl8.6
D:/绿色软件/library
D:/library
D:/tcl8.6.9/library
D:/tcl8.6.9/library

网上找了一圈没有解决,有熟悉的没?

**Comment by hiroi-sora** [comment, marked as answer]
确实比较奇怪，看起来是tk库的文件缺失。但所有库文件是压缩存储在exe里，运行时才解压释放到TEMP临时目录，也就是不管怎么清理临时目录也不会影响到下一次运行才对。

以下是一些可能的解决方法：
1. 检查是否存在目录 C:/Windows/TEMP ，如果不存在，则手动创建。然后启动Umi尝试。
2. 如果还不行，尝试重新下载 Umi v1.3 。
3. 或者，你也可以尝试 [UmiOCR v2.0](https://github.com/hiroi-sora/Umi-OCR_v2) 预览版，它采用与1.3完全不同的启动机制，可能可以绕过环境的影响。

**Reply by BasicSix** [reply]
谢谢亲.后来我把软件启动模式改为管理员,可以运行了,但还是开机不能启动.很有意思的情况.
